README.md

```diff
@@ -21,15 +21,15 @@ The add-on depends on the Kodi add-on "requests" for the ambilight mode.
 **Kodi add-on script.module.requests**
 
  - Download the add-on as a ZIP file from https://github.com/beenje/script.module.requests
-  - (Right click on the "ZIP" icon and select "Download Linked File").
+  - (Click on the green "Clone or download button" then right click on the "Download ZIP" link and select "Download Linked File").
```

Comment:
I don't think the "Download Linked File" part is still relevant? I did a quick check as I usually only care about git part, got the file after clicking "Download ZIP".